kubernetes/resource_kubernetes_env.go

```diff
@@ -324,15 +335,26 @@ func getManagedEnvs(managedFields []v1.ManagedFieldsEntry, manager string, d *sc
 		if err != nil {
 			return nil, err
 		}
-		spec := mm["f:spec"].(map[string]interface{})
-		template := spec["f:template"].(map[string]interface{})
-		templateSpec := template["f:spec"].(map[string]interface{})
-		containers := templateSpec["f:containers"].(map[string]interface{})
+
+		spec, _, err := unstructured.NestedMap(u.Object, "spec", "template", "spec")
```

Comment:
Shouldn't these be prefixed with `f:` ?

Reply:
Yep! Seems like the reason for my issues on friday